netbox/ipam/utils.py

```diff
@@ -164,11 +164,11 @@ def available_vlans_from_range(vlans, vlan_group, vid_range):
         prev_vid = vlan.vid
 
     # Annotate any remaining available VLANs
-    if prev_vid < max_vid:
+    if prev_vid < max_vid and (available := max_vid - prev_vid - 1) > 0:
```

Comment:
While this works, it's a bit confusing IMO. Would it not be sufficient to just account for the off-by-one error (and leave the value of `available` as is)?

```suggestion
    if prev_vid < max_vid - 1:
```

Reply:
Fair point! I must have had some tunnel vision going on and didn't see the simpler, cleaner fix. 

I'll revise it.